packages/blade/src/components/Accordion/AccordionItem.tsx

```diff
@@ -40,37 +59,29 @@ type AccordionItemProps = {
   _index?: number;
 } & TestID;
 
-const BLANK_SPACE = ' ';
-
-/**
- * On React Native if the `AccordionItem` has a lengthy description which renders a `Text` spanning multiple lines,
- * it sometimes messes up the layout calculation (it thinks of multiline as a single line before flowing in the UI).
- * And during the expanding / collapsing animation, text reflows causing words to jump around across lines.
- *
- * Rendering a blank `Text` at the end seems to fix all this 🤯
- */
-const reactNativeMultilineTextOverflowFix = (
-  // make this hidden from screen readers
-  <BaseBox {...makeAccessible({ hidden: true })}>
-    <Text>{BLANK_SPACE}</Text>
-  </BaseBox>
-);
-
 const AccordionItem = ({
   title,
   description,
   icon,
   children,
+  isDisabled,
   _index,
   testID,
 }: AccordionItemProps): ReactElement => {
-  const { expandedIndex, onExpandChange, defaultExpandedIndex } = useAccordion();
+  const {
+    expandedIndex,
+    onExpandChange,
+    defaultExpandedIndex,
+    variant,
+    numberOfItems,
+  } = useAccordion();
   const isExpanded = expandedIndex === _index;
   const isDefaultExpanded = defaultExpandedIndex === _index;
+  const isDeprecatedAPI = Boolean(title) || Boolean(description);
+  const [header, body] = React.Children.toArray(children);
```

Comment:
we can't until we remove deprecated API. Because passing anything is valid in the old API. People can pass any child and that becomes the content slot for old API.

```jsx
<AccordionItem>
  <div>Random content</div>
</AccordionItem>
```

Reply:
oh right! the checks you have added should be sufficient 👍